### PR TITLE
Enhancement for issue #307

### PR DESCRIPTION
**Pull request by alexsasharegan**
Support for text alignment classes on the form input label per issue #307.

Didn't see a test runner in the package.json besides linting, so I assume circle-ci will test this upon submission.

### CODE REVIEW

File: lib/components/form-fieldset.vue [hunk 1]
```diff
@@ -68,6 +79,11 @@
                 type: Number,
                 default: 3
             },
+            labelTextAlign: {
+                type: String,
+                required: false,
+                default: 'left'
```
Comment (pi0) [inline lib/components/form-fieldset.vue]: Please don't make left default, it breaks global alignment in RTL projects.

File: lib/components/form-fieldset.vue [hunk 2]
```diff
@@ -35,6 +35,17 @@
...
             },
+            labelAlignClass() {
+                switch (this.labelTextAlign.toLowerCase()) {
+                    case 'center':
+                        return 'text-center';
+                    case 'right':
+                        return 'text-right';
+                    case 'left':
+                    default:
+                        return 'text-left';
```
Comment (pi0) [inline lib/components/form-fieldset.vue]: Also default would be just null :)

**Comment by pi0** [comment]
Thanks!

**Comment by alexsasharegan** [comment]
Cheers! 🍻

File: lib/components/form-fieldset.vue (hunk 1 above)
Comment (mosinve) [inline lib/components/form-fieldset.vue]: Null сauses "TypeError: Cannot read property 'toLowerCase' of null" when labelTextAlign isn't specified at component level.....

**Comment by mosinve** [comment]
This PR is a breaking change to my project.

**Pull request merged**